### PR TITLE
Running python from java

### PR DESCRIPTION
**Pull request by elena-lyulina**

### CODE REVIEW

File: src/test/kotlin/org/jetbrains/research/transformations/PythonTransformationSetupTest.kt [hunk 1]
```diff
@@ -25,7 +26,7 @@ internal class PythonTransformationSetupTest: PythonTransformationsTest() {
     fun checkInverseParserSetup() {
         val treeCtx = getTreeContext()
         // Todo: Find a better way?
-//        println("code: ${getSourcePythonCode(treeCtx, XMLTreeFileName)}")
+        println("code: ${getSourcePythonCode(treeCtx, XMLTreeFileName)}")
```
Comment (nbirillo) [inline src/test/kotlin/org/jetbrains/research/transformations/PythonTransformationSetupTest.kt]: Should we delete this `println`? Or at least replace it into a logger?

File: src/test/kotlin/org/jetbrains/research/transformations/util/ParserSetup.kt [hunk 2]
```diff
@@ -20,6 +21,9 @@ import org.jetbrains.research.transformations.util.Util.runProcessBuilder
 object ParserSetup {
     private val LOG = Logger.getLogger(javaClass.name)
 
+    private const val PYTHON3_PROPERTY = "ac.p3.path"
+    private const val PYTHONPARSER_PROPERY = "gt.pp.path"
```
Comment (nbirillo) [inline src/test/kotlin/org/jetbrains/research/transformations/util/ParserSetup.kt]: PROPERY -> PROPERTY

File: src/test/kotlin/org/jetbrains/research/transformations/util/Util.kt [hunk 3]
```diff
@@ -7,17 +7,18 @@ import java.io.InputStreamReader
 
 object Util {
 
+    data class Command(val command: List<String>, val directory: String? = null, val variables: Map<String, String>? = null)
```
Comment (nbirillo) [inline src/test/kotlin/org/jetbrains/research/transformations/util/Util.kt]: Please, add a comment on what is **variables** and **directory** or rename them

**Review by nbirillo — CHANGES_REQUESTED** [review]
Great job, thanks! But I have several remarks. Also, fix the branches conflicts, please

File: src/test/kotlin/org/jetbrains/research/transformations/util/ParserSetup.kt (hunk 2 above)
Comment (elena-lyulina) [reply src/test/kotlin/org/jetbrains/research/transformations/util/ParserSetup.kt]: thanks!

**Review by nbirillo — APPROVED** [review]
Looks well!

**Pull request merged**